src/wui/info_panel.cc

```diff
@@ -383,7 +383,7 @@ void InfoPanel::set_fps_string(const bool show,
 		text_fps_.set_text("");
 		text_fps_.set_tooltip("");
 	} else {
-		const std::string text = format("%5.1f fps (avg: %5.1f fps)", fps, average);
+		const std::string text = format("%05.1f fps (avg: %05.1f fps)", fps, average);
```

Comment:
(I think this would have been better in a separate PR, though both changes are simple, so no big harm)

As this is 1.2 material, I think `base/format/{number,float}_node.h` should be fixed to use `kDigitWidthSpace` for padding in localised mode, and use `format_l()` instead of padding with zeroes.

(… except this isn't localised, but why? I've just tested it and it has no problem with unicode chars.)

Reply:
Sorry, I assumed the change was trivially acceptable. I can look into it separately.